docs/content/docs/recipes/collaborative-text-area.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Building a collaborative TextArea
+menuPosition: 4
+author: scottn12
+---
+
+In this tutorial, you'll learn how to use the [SharedString]({{< relref "string.md" >}}) distributed data structure (DDS) with [React](https://reactjs.org/) to create a collaborative text area. SharedString is a DDS with specialized features and behaviors for working with text.
+
+To jump ahead into the finished demo, check out the [SharedString example in our FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/collaborative-text-area).
+
+The following image shows a textarea open in four browsers. The same text is in all four.
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/collaborative_text_area_1.png)
+
+The next image shows the same four clients after an edit was made in one of the browsers. Note that the text has updated in all four browsers.
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](https://fluidframework.blob.core.windows.net/static/images/collaborative_text_area_2.png)
```

Comment:
how do these images look on the website?  can't check right now because it's complaining about broken links, but looking at the images themselves it may be better to shrink the window sizes since there's a lot of empty space, which also makes it harder to tell the difference between the before/after

Reply:
Right now the text is bit small, so i think I will redo these screenshots